its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/IncrementalPRAnalysisSonarCloudTest.java

```diff
@@ -159,7 +159,7 @@ private static void runBeginStep(Path projectDir, StreamConsumer.Pipe logsConsum
       .addArgument("/o:" + SONARCLOUD_ORGANIZATION)
       .addArgument("/k:" + SONARCLOUD_PROJECT_KEY)
       .addArgument("/d:sonar.host.url=" + SONARCLOUD_URL)
-      .addArgument("/d:sonar.login=" + SONARCLOUD_PROJECT_TOKEN)
+      .addArgument("/d:sonar.login=" + SONARCLOUD_PROJECT_TOKEN) // SonarCloud does not support yet sonar.token
```

Comment:
We cannot change these for now since SonarCloud is not supporting the property.

Reply:
Could you create a ticket specifically for this test, to not forget about changing it after they support it?

Reply:
Ah yeah you are right, the tests are supposed to fail